pkg/log/log.go

```diff
@@ -0,0 +1,200 @@
+package log
```

Comment:
Why is this needed?

Reply:
This is so that we can log the retries if the rollup controller autoscalling API calls get throttled. Similar to the way it was implemented in lifecycle manager https://github.com/keikoproj/lifecycle-manager/blob/7661ea7bd6fc4f5ddd92243252500a8e634ebc57/cmd/serve.go#L156-L159